src/sections/Home/Banner/index.js

```diff
@@ -1,24 +1,25 @@
 import React, { useEffect, useState } from "react";
 import BannerDefault from "../Banner-default";
-// import BannerAlt from "../Banner-alt"; 
+// import BannerAlt from "../Banner-alt";
 
 const Banners = [
   <BannerDefault key={0} />,
-  // <BannerAlt key={1}/> 
+  // <BannerAlt key={1} /> 
 ];
 
 const Banner = () => {
-  const [Banner, setBanner] = useState(0);
+  let initialValue;
+  try {
+    initialValue = sessionStorage.getItem("banner") || 0;
```

Comment:
If we are doing a `OR` here, do we still need to wrap it in a try-catch block?

Reply:
sessionStorage is not defined during server side builds. So builds and CI wont pass. So wrapping it in a try-catch block.

Reply:
Okay